package.json

```diff
@@ -20,8 +20,10 @@
     "test": "node_modules/.bin/mocha",
     "lint-staged": "node_modules/.bin/lint-staged",
     "coverage-report": "node --loader ts-node/esm util/gen_coverage_report.js",
-    "find-translations": "node --loader ts-node/esm util/find_missing_translations.js",
-    "translate-timeline": "node --loader ts-node/esm util/translate_timeline.ts",
+    "util": "node --loader ts-node/esm util/index.ts",
+    "find-translations": "npm run util -- findTranslations",
```

Comment:
`npm run find-translations -- -l fr -f 05-shb` finds quite a few translations before this patch and nothing with this patch.

Reply:
Should be fixed, can you try again?

Reply:
Seems fixed, thanks!